nx.json

```
@@ -0,0 +1,43 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": [
+          "compile",
+          "lint",
+          "version:update"
+        ]
+      }
+    }
+  },
+  "targetDefaults": {
+    "compile": {
+      "dependsOn": [
+        "version:update",
+        "^compile"
+      ],
+      "inputs": [
+        "{projectRoot}/src",
+        "{projectRoot}/test"
+      ],
+      "outputs": [
+        "{projectRoot}/build"
+      ]
+    },
+    "lint": {
+      "inputs": [
+        "{projectRoot}/src",
+        "{projectRoot}/test"
+      ]
+    },
+    "version:update": {
+      "inputs": [
+        "{projectRoot}/package.json"
+      ],
+      "outputs": [
+        "{projectRoot}/src/version.ts"
+      ]
+    }
+  }
+}
```

nit: missing newline
```suggestion
}

```